protos/sigstore_rekor.proto

```diff
@@ -110,4 +110,9 @@ message TransparencyLogEntry {
         // entry was appended to the log, and that the log has not been
         // altered.
         InclusionProof inclusion_proof = 6;
+        // The canonicalized Rekor entry body, used for SET verification. This
+        // is the same as the body returned by Rekor. It's included here for
+        // cases where the client cannot deterministically reconstruct the
+        // bundle from the other fields.
```

Comment:
nit: per previous convo and Zack's comment.. add 
"Clients MUST verify that the signature referenced in the canonicalized_body matches the signature provided in the bundle content."